packages/base-driver/test/unit/wd-proxy/mock-request.js

```diff
@@ -0,0 +1,38 @@
+function resFixture(url, method) {
```

Comment:
fwiw, I think `sinon` could do this, but maybe it's not helpful to do so in this context

Reply:
meh